wremnants/datasets/datagroups.py

```diff
@@ -368,20 +399,40 @@ def __init__(self, infile, combine=False, pseudodata_pdfset = None, applySelecti
                     selectOp = sigOp,
                 ), 
                 "Fake" : dict(
-                    members = list(self.datasets.values()),
+                    members = list(filter(lambda y: y.group != "QCD", self.datasets.values())),
                     scale = lambda x: 1. if x.is_data else -1,
                     label = "Nonprompt",
                     color = "grey",
                     selectOp = fakeOp,
                 ),
+                "QCD" : dict(
+                    members = list(filter(lambda y: y.group == "QCD", self.datasets.values())),
+                    label = "QCD MC",
+                    color = "grey",
+                    selectOp = sigOp,
+                ), 
+           
             })
         else:
             self.groups["Other"] = dict(
-                members = [x for x in self.datasets.values() if not x.is_data and x.name not in ["ZmumuPostVFP", "ZtautauPostVFP"]],
+                members = [x for x in self.datasets.values() if not x.is_data and x.name not in ["ZmumuPostVFP", "ZtautauPostVFP"] and x.group != "QCD"],
```

Comment:
Why check the group attribute instead of the name?

Reply:
Mainly because the actual sample name tends to be longer and less trivial (this is our choice, but it is like that to resemble the actual name of the dataset in DAS, which I think makes sense). So then if one wants to select/exclude samples at run time it is simpler to act on groups since names are shorter (and this would allow us to select all processes associated to that group, which I guess is what one wants, it probably wouldn't make sense to exclude only a specific dataset belonging to the group).

This could be extended to really allow to manipulate single data set instead of global group, but it makes the code more complicated to catch all possible cases, and it goes beyond the scope of the select/exclude feature, which was really meant to exclude things as a whole